src/commands/remoteDebug/getRemoteDebugLanguage.ts

```diff
@@ -6,14 +6,17 @@
 import { SiteConfig } from '@azure/arm-appservice';
 import { RemoteDebugLanguage } from '@microsoft/vscode-azext-azureappservice';
 
-export function getRemoteDebugLanguage(siteConfig: SiteConfig): RemoteDebugLanguage {
+export function getRemoteDebugLanguage(siteConfig: SiteConfig, appServicePlan?: string): RemoteDebugLanguage {
     // Read siteConfig.linuxFxVersion to determine debugging support
     //   If the Function App is running on Windows, it will be empty
-    //   If the Function App is running on Linux consumption, it will be empty
+    //   If the Function App is running on Linux consumption plan remote debugging is not supported
     //   If the Function App is running on a Linux App Service plan, it will contain Docker registry information, e.g. "DOCKER|repo.azurecr.io/image:tag"
-    //      The blessed Node.js Docker image will be something like "DOCKER|mcr.microsoft.com/azure-functions/node:2.0-node8-appservice"
-    if (siteConfig.linuxFxVersion && siteConfig.linuxFxVersion.startsWith('DOCKER|mcr.microsoft.com/azure-functions/node')) {
-        return RemoteDebugLanguage.Node;
+    //      or it will contain language information, e.g. "Node|14"
+    const isNotConsumption: boolean = appServicePlan?.toLowerCase() !== 'y';
+    if (isNotConsumption) {
```

Comment:
I think we can upgrade this to an case-insensitive regex expression. I know you're up for the task 👩‍🎓

Reply:
![image](https://user-images.githubusercontent.com/59709511/209394225-38dc8bec-fc5e-430a-a59b-3e050daef7fb.png)